tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php

```diff
@@ -38,7 +38,6 @@ class GoogleAdsErrorsTest extends TestCase
     public function setUp()
     {
         $this->failure = $this->createGoogleAdsFailure();
-
```

Comment:
Could you revert this change to be consistent with V1?

Reply:
Actually, I removed this line because it was not consistent with V1 (see [here](https://github.com/googleads/google-ads-php/blob/master/tests/Google/Ads/GoogleAds/Util/V1/GoogleAdsErrorsTest.php#L41))